Finish mobile version

FredsephT32 opened the pull request:
- The  Works section with the 4 cards and their respective image is completed.
- The About Myself section with the List and Images completed.

FredsephT32 committed:
Mobile-Project-Continuation

Added more content to my index.html, the 4 cards integrated and personalized with css.

FredsephT32 committed:
added the About My Self Section

there is still a few things to add and fix.

FredsephT32 committed:
Added the last couple of Images

the Images for the arrows and languages bars such as the jss, html, css and also fixed some erros with the code.

FredsephT32 committed:
Image link error solved

Images in the last part of the project below the "About me" section are now working properly.

FredsephT32 committed:
Linter Webhint error

Lines 168, 173, 175 changed

mgmediaweb commented on index.html:
```diff
@@ -20,7 +20,6 @@
 
            <h2 class="glad-txt">I'm Frederick Glad to see you!</h2>
```
Kindly, this should be your main title, I recommend you use the H1 tag, it is better for the accessibility of your site

mgmediaweb commented on index.html:
```diff
@@ -44,5 +43,140 @@ <h3 class="small-txt">LET'S CONNECT</h3>
 
    </header> 
```
Kindly, by convention the HEADER tag is used to contain the top bar and within it find the NAVBAR, I recommend you make this change so that your site is better structured.

mgmediaweb commented on index.html:
```diff
@@ -44,5 +43,140 @@ <h3 class="small-txt">LET'S CONNECT</h3>
+    <div class="list-container"> 
+
+      <ul><li class="lang-label"> <h3>Languages</h3> <img src="Images/Arrow_down.svg" alt="Arrow_down"></li>
+        <li><img src="Images/JavaScript-bar.png" alt="javaScript"></li> 
+        <li><img src="Images/HTML-bar.png" alt="HTML"></li>
+        <li><img src="Images/CSS-bar.png" alt="CSS"></li>
```
Kindly, I recommend that all these languages are HTML code and only use the images for the logo.

mgmediaweb reviewed (changes requested):
Hi @FredsephT32 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- Kindly, could you remove the margin-left to make it look more like the Figma design

![image](https://user-images.githubusercontent.com/41912912/185529021-d69e3d2c-06af-46c9-9c11-63305e8a347c.png)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FredsephT32 committed:
Code Review Changes

changes on the structure of the Mobile Skeleton.

- Header is now in the right place.
- Prog Lngs Images are just only the Icon.

FredsephT32 committed:
Linter fails solved

Webhint and Stylelint minor errors fixed

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @FredsephT32 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - GitHub flow is followed. ✔️
 - Professional README file. ✔️
 - PR has a good title and summary. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
